project/vision_backend/tasks.py

```diff
@@ -114,8 +114,8 @@ def submit_classifier(source_id, nbr_images=1e5, force=False):
     # Create val-labels
     val_labels = th.make_dataset([image for image in images if image.valset])
 
-    # This will not include the one we just created, b/c it is not valid.
-    prev_classifiers = Classifier.objects.filter(source=source, valid=True)
+    # This will not include the one we just created, b/c status isn't accepted.
+    prev_classifiers = source.get_accepted_robots()
```

Comment:
@beijbom From your memory, is `prev_classifiers` meant to have the classifiers in a particular order? I changed it to list classifiers from newest to oldest. The old order was arbitrary, as far as I could tell, but I just wanted to double check.

Reply:
You are right in that it doesn't matter. We just compare the current accuracy to max across all the previous ones.